charts/stardog/values.yaml

```diff
@@ -41,7 +41,11 @@ ports:
   server: 5820
   # sql: 5806
 
-# The location Stardog will use for temporary space (i.e. java.io.tmpdir)
+# The location Stardog will use for temporary space (i.e. java.io.tmpdir).
+# if you choose to place this in STARDOG_HOME (/var/opt/stardog) for performance
+# reasons, insure that it does not conflict with any possible database names.
+# For example, a good choice might be /var/opt/stardog/tmp-4646E7B662A7.
+# If the directory does not exist it will be created.
```

Comment:
Can we also put this similar warning in the README with tmpDir:
https://github.com/stardog-union/helm-charts/blob/develop/charts/stardog/README.md

Reply:
Good call.